ssh/DOCS.md

```diff
@@ -249,6 +249,29 @@ based on the following:
 - `MINOR`: Backwards-compatible new features and enhancements.
 - `PATCH`: Backwards-compatible bugfixes and package updates.
 
+## Visual Studio Code Remote - SSH
+
+Some customization is required prior to being able to connect to your Home 
+Assistant install with Visual Studio Code. The following configuration will aid
+in setting up VSCode Remote - SSH:
+
+```yaml
+packages:
+  - gcompat
+  - libstdc++
+  - curl
+allow_remote_port_forwarding: true
+allow_tcp_forwarding: true
+```
+
+### Notes
+* Allowing remote port and tcp forwarding lower the overall level of security of 
+your SSH server. Nevertheless, this warning is debatable. 
```

Comment:
Let's not debate it in the documentation.

Reply:
It's useful to note that the security will be lowered, but I can remove the last sentence there